{"name": "test_create_user_with_incorrect_company_id_021", "status": "passed", "description": "\n    Зарегистрировать нового пользователя с не верным company_id\n\n    Ожидаемый результат:\n        Статус-код 404;\n        Время ответа сервера - не превышает 500ms;\n        Схема JSON-ответа соответствует Требованиям;\n        Response header \"Content-Type\": \"application/json\"\n        Response header \"Connection\": \"keep-alive\"\n        В JSON - присутствует ключ detail, значением является описание ошибки\n        В тексте ошибки указан отправленный нами \"company_id\"\n    ", "start": 1721415264106, "stop": 1721415264146, "uuid": "67c6d16d-8756-47ec-b1f3-7e95473accce", "historyId": "c25d56aa6ea3b9ec280628ab81aed2f6", "testCaseId": "c25d56aa6ea3b9ec280628ab81aed2f6", "fullName": "tests.users.test_users#test_create_user_with_incorrect_company_id_021", "labels": [{"name": "tag", "value": "users"}, {"name": "parentSuite", "value": "tests.users"}, {"name": "suite", "value": "test_users"}, {"name": "host", "value": "DESKTOP-55MLA3F"}, {"name": "thread", "value": "23412-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.users.test_users"}]}